handyman/data/product_pricelist.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="product_pricelist_1" model="product.pricelist">
```

Comment:
This is created by default when activating the pricelist in the setting. You can remove (and links to it in SO and partners as well)